Content.Server/GameObjects/EntitySystems/CargoConsoleSystem.cs

```diff
@@ -138,7 +138,7 @@ public bool ChangeBalance(int id, int amount)
 
         public bool AddOrder(int id, string requester, string reason, string productId, int amount, int payingAccountId)
         {
-            if (amount < 1 || !TryGetOrderDatabase(id, out var database))
+            if (amount < 1 || !TryGetOrderDatabase(id, out var database) || amount > database.MaxOrderSize)
```

Comment:
This is the right way to go about this, but could you also add tracking of total items ordered to check against?

Reply:
You mean limiting the amount of requested crates to the max order size?

Reply:
I mean that since you're checking `amount > database.MaxOrderSize` someone could get around it by ordering a bunch of items with amounts of `database.MaxOrderSize - 1` and they'd be able to order more crates than the max that way.

Reply:
I gotcha. Yeah no problem.

Reply:
why did you resolve this? afaict you haven't done it

Reply:
So I put this in the wrong place originally, this piece of code is for requesting orders, which means it should be fine. Approving orders now includes the total quantity check.